backend/sequelizeManager.js

```diff
@@ -90,31 +90,56 @@ export class SequelizeManager {
     }
 
     createConnection(configuration) {
-
-        const {
+        let {
             username, password, database, port,
             dialect, storage, host
             } = configuration;
 
-        this.log(`Creating a connection for user ${username}`, 1);
-
-        this.connection = new Sequelize(database, username, password, {
+        let options = {
             dialect,
             host,
             port,
             storage
-        });
+        };
+
+        this.log(`Creating a connection for user ${username}`, 1);
+
+        let subDialect = null;
+        if (this.connection) {
+            subDialect = this.connection.config.subDialect || null;
+        }
```

Comment:
if connection exists, check if it has a `subDialect` parameter and save it to a local `subDialect` variable to use afterwards.